roles/common/tasks/monitoring.yml

```diff
@@ -191,9 +191,14 @@
 
 - name: check for static route on multi subnet
   sensu_check: name=check-static-route-{{ item.cidr | regex_replace('(?P<host>.+)/(?P<size>.+)', '\\g<host>_\\g<size>') }} plugin=check-static-route.py
-               args='--subnet {{ item.cidr }} --gateway {{ item.gateway }} --criticality {{ common.monitoring.sensu_checks.check_static_route.criticality}}'
+               args='--subnet {{ item.cidr }} --gateway {{ item.gateway }} --criticality {{ common.monitoring.sensu_checks.check_static_route.criticality }}'
   with_items: "{{ undercloud_cidr }}"
-  when: "{{undercloud_cidr is defined and undercloud_cidr | length > 1}}"
+  when: "{{ undercloud_cidr is defined and undercloud_cidr | length > 1 }}"
+  notify: restart sensu-client
+
+- name: check that large receive offload (LRO) is off
+  sensu_check: name=check-large-receive-offload plugin=check-large-receive-offload.py
+               args="--devices r\'"{{ hostvars[inventory_hostname].ansible_bond0.slaves }}"\' --criticality {{ common.monitoring.sensu_checks.check_large_receive_offload.criticality }}"
   notify: restart sensu-client
```

Comment:
I think this should be gated off `when: common.monitoring.sensu_checks.check_lro.enabled|bool`